### PR TITLE
v0.35.0

### PR DESCRIPTION
**Pull request by RomainMuller**
See CHANGELOG

----

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,176 @@
...
+* **stepfunctions:** add grantStartExecution() ([#2793](https://github.com/awslabs/aws-cdk/issues/2793)) ([da32176](https://github.com/awslabs/aws-cdk/commit/da32176))
+* **stepfunctions:** add support for AmazonSageMaker APIs ([#2808](https://github.com/awslabs/aws-cdk/issues/2808)) ([8b1f3ed](https://github.com/awslabs/aws-cdk/commit/8b1f3ed)), closes [#1314](https://github.com/awslabs/aws-cdk/issues/1314)
+* **stepfunctions:** waitForTaskToken for Lambda, SQS, SNS ([#2686](https://github.com/awslabs/aws-cdk/issues/2686)) ([d017a14](https://github.com/awslabs/aws-cdk/commit/d017a14)), closes [#2658](https://github.com/awslabs/aws-cdk/issues/2658) [#2735](https://github.com/awslabs/aws-cdk/issues/2735)
+* formalize the concept of physical names, and use them for cross-environment CodePipelines. ([#1924](https://github.com/awslabs/aws-cdk/issues/1924)) ([6daaca8](https://github.com/awslabs/aws-cdk/commit/6daaca8))
+
+
+*  refactor(events): clean up Events APIs (#2840) ([1e23921](https://github.com/awslabs/aws-cdk/commit/1e23921)), closes [#2840](https://github.com/awslabs/aws-cdk/issues/2840)
```
Comment (eladb) [inline CHANGELOG.md]: These look like they are in the wrong section

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,176 @@
...
+* **core:** `stack.missingContext` is now private
+* **core:** `stack.renameLogical` have been renamed to `stack.renameLogicalId`
+* **core:** `IAddressingScheme`, `HashedAddressingScheme` and `LogicalIDs` are now internal. Override `Stack.allocateLogicalId` to customize how logical IDs are allocated to resources.
+* **cli:** The CLI no longer accepts `--rename`, and the stack
+                 names are now immutable on the stack artifact.
+* **sns:** using a queue, lambda, email, URL as SNS Subscriber now
+requires an integration object from the `@aws-cdk/aws-sns-subscribers`
+package.
+* **ecs-patterns:** Renamed QueueWorkerService for base, ec2 and fargate to QueueProcessingService, QueueProcessingEc2Service, and QueueProcessingFargateService.
+* * iam: `roleName` in `RoleProps` is now of type `PhysicalName`
```
Comment (eladb) [inline CHANGELOG.md]: Formatting here looks off

File: CHANGELOG.md (hunk 1 above)
Comment (RomainMuller) [reply CHANGELOG.md]: Yeah I figured, moved them into *refactor* section and polished name up.

File: CHANGELOG.md (hunk 2 above)
Comment (RomainMuller) [reply CHANGELOG.md]: Fixed. not sure what happened there 🤷🏻‍♂️

**Pull request merged**